src/tests/WebApi.MsBuild.SystemTest.ValidateContract/WebApi.MsBuild.SystemTest.ValidateContract.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <IsPackable>false</IsPackable>
+        <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
+        <RootNamespace>WebApi.MsBuild.SystemTest</RootNamespace>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <OpenApiDevelopmentMode>GenerateContract</OpenApiDevelopmentMode>
```

Comment:
Is it supposed to be ValidateContract here?

Reply:
Fixed